src/lightning_app/utilities/cli_helpers.py

```diff
@@ -119,3 +121,16 @@ def _retrieve_application_url_and_available_commands(app_id_or_name_or_url: Opti
                     raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
                 return lightningapp.status.url, _extract_command_from_openapi(resp.json())
     return None, None
+
+
+def _arrow_time_callback(
+    _ctx: "click.core.Context", _param: "click.core.Option", value: str, arw_now=arrow.utcnow()
+) -> arrow.Arrow:
+    try:
+        return arw_now.dehumanize(value)
+    except ValueError:
+        raise click.ClickException(f"cannot parse time {value}")
+    try:
```

Comment:
this code is unreachable

Reply:
this callback is very limited to human friendly formats, e.g. `2 days ago` , `in a month` but won't work in case of a normal timestamp. `Mon Sep 08 16:41:45 2014` .

You can just parse the value in case of an exception.

Reply:
Added a possibility of parsing ISO-8601 (default in `arrow.get`)
To parse `Mon Sep 08 16:41:45 2014` we have to have a bit more of logic, not sure we need it.